test/jit/test_with.py

```diff
@@ -426,15 +432,18 @@ def test_exception_fn_call(x, c):
 
         c = Context(1)
 
-        with self.assertRaises(Exception):
+        # checkScript and checkScriptRaisesRegex cannot be used because the string frontend will
+        # not compile class types (of which Context, the context manager being used for this test
+        # is one).
+        with self.assertRaisesRegex(Exception, r"raised exception"):
             test_exception(torch.randn(2), c)
         self.assertEqual(c.count, 1)
 
-        with self.assertRaises(Exception):
+        with self.assertRaisesRegex(Exception, r"raised exception"):
```

Comment:
Should we use assertRaisesRegexWithHighlight in here?

Reply:
There is nothing to highlight; the code compiles without any problems but throws and exception by design.